google/cloud/pubsub_v1/publisher/_sequencer/base.py

```diff
@@ -45,18 +47,17 @@ def unpause(self, message):  # pragma: NO COVER
 
     @staticmethod
     @abc.abstractmethod
-    def publish(self, message, retry=None, timeout=None):  # pragma: NO COVER
+    def publish(
+        self, message, retry=None, timeout: gapic_types.TimeoutType = None
```

Comment:
Can we use `DEFAULT` rather than `None`?  The implementation defaults to `DEFAULT`.

Reply:
That makes sense, the ABC should be consistent.